libs/csm/rest/csm_rest_query_deployment.py

```diff
@@ -216,3 +206,30 @@ def verify_storage_set(self, cluster_id: str, storage_set_id: str = None,
             self.log.error(err_msg)
             result = False
         return resp, result, err_msg
+
+    def verify_error_message(self, resp, resp_error_code, resp_msg_id, resp_msg_index):
```

Comment:
Can u move this to csm_rest_test_lib.py file

Reply:
and we can use this for all tests at some point

Reply:
moved to csm_rest_test_lib